kubernetes/kube-state-metrics-service.yaml

```diff
@@ -1,15 +1,16 @@
 apiVersion: v1
 kind: Service
 metadata:
-  annotations:
-    prometheus.io/scrape: 'true'
```

Comment:
True, I'll add this back. It seems that the community has moved away from this use, but if not present it may confuse users who do expect it.